engine/gamesys/src/gamesys/test/test_gamesys.cpp

```diff
@@ -123,7 +123,7 @@ static dmGameObject::HInstance Spawn(dmResource::HFactory factory, dmGameObject:
 
 static dmGameObject::HInstance Spawn(dmResource::HFactory factory, dmGameObject::HCollection collection, const char* prototype_name, dmhash_t id)
 {
-    return Spawn(factory, collection, prototype_name, id, 0, 0, Point3(0, 0, 0), Quat(0, 0, 0, 1), Vector3(1, 1, 1));
+    return Spawn(factory, collection, prototype_name, id, 0, Point3(0, 0, 0), Quat(0, 0, 0, 1), Vector3(1, 1, 1));
```

Comment:
is this using the new spawn function introduced in this PR?

Reply:
This is the local test_gamesys.cpp:Spawn (see above), and it in turn uses the new dmGameObject::Spawn which takes a HProperties.